src/lightning-mocks/accordion/accordion.js

```diff
@@ -8,6 +8,5 @@ import { LightningElement, api } from 'lwc';
 
 export default class Accordion extends LightningElement {
     @api activeSectionName
-	@api allowMultipleSectionsOpen
-
-}
+    @api allowMultipleSectionsOpen
+}
```

Comment:
Not sure what happened in this diff since line 11 didn't change at all but looks ok I guess.